android-auto-app/src/main/res/xml/automotive_app_desc.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<automotiveApp>
+    <uses name="template" />
+</automotiveApp>
```

Comment:
This should not be needed because it is in the SDK right? 

https://github.com/mapbox/mapbox-navigation-android/blob/6b92019c678b1777b310c0ef362633c11847ed9d/libnavui-androidauto/src/main/res/xml/automotive_app_desc.xml#L1-L5

We don't have it in our test app 🤔 . Did this fix an issue?

Reply:
Thanks @kmadsen! I didn't realized it was in the SDK. Somehow I saw Android Studio complaining on my end:
![image](https://user-images.githubusercontent.com/914983/210529252-213c48ee-89c6-4a29-a5f9-a02193987563.png)

Reply:
Oh I see, that could probably be resolved with a gradle sync